client/src/pages/private/ParticipantTable.js

```diff
@@ -733,6 +771,13 @@ export default () => {
                   if (columnId === 'status') {
                     return prettifyStatus(row[columnId], row.id, tabValue, handleEngage);
                   }
+                  if (columnId === 'distance') {
+                    if (row[columnId] !== null && row[columnId] !== undefined) {
+                      return `${math.round(row[columnId]/1000)|| "<1"} Km`;
+                    }
+                    
+                    return 'N/A';
```

Comment:
I don't know why our linter is so much looser on the FE
```suggestion
                      return `${math.round(row[columnId]/1000) || '<1'} Km`;
                    }
                    return 'N/A';
```

Reply:
Preferring single quotes here?